models/general/knowledge/en/number_of_galaxies.txt

```diff
@@ -0,0 +1,23 @@
+How many galaxies are in this universe| Galaxies in universe | Number of galaxies in universe | Number of galaxies 
```

Comment:
Make this `How many galaxies are there in this universe| Galaxies in the universe | Number of galaxies in the universe | Number of galaxies`

I suppose same goes for the other two skills as well?